aries_cloudagent/multitenant/manager.py

```diff
@@ -124,6 +124,17 @@ async def get_wallet_profile(
                 "wallet.type": None,
             }
 
+            dispatch_type = extra_settings.get("wallet.dispatch_type")
```

Comment:
I think we don't need to pass extra_settings for `wallet.webhook_urls` and `wallet.dispatch_type` in here, because we already have these values in `wallet_record.wallet.webhook_urls` and `wallet_record.wallet.dispatch_type`.